src/battle_script_commands.c

```diff
@@ -1021,16 +1074,16 @@ static const u16 sMoveEffectsForbiddenToInstruct[] =
 
```

Comment:
Here are some corrections for this table:
```
Instruct Forbidden Missing:
EFFECT_CONFUSE_HIT (for Chatter)
EFFECT_RECOIL_25 (for Struggle)
EFFECT_UPROAR (for Uproar)

Instruct Forbidden Incorrect:
EFFECT_RECHARGE
```

Also because some moves are incorrectly affected because we use move effects for Instruct for some reason and some moves share move effects with moves that are forbidden for Instruct. So I think we should eventually add Instruct Forbidden to the other forbidden array if possible.

One example is `EFFECT_DO_NOTHING` which is meant to affect Celebrate and Hold Hands, but it it also incorrectly affects Splash and Happy Hour.